cloud_experts_tutorials/cloud-experts-using-alb-and-waf.adoc

```diff
@@ -0,0 +1,443 @@
+:_content-type: ASSEMBLY
+[id="cloud-experts-using-alb-and-waf"]
+= Tutorial: Using AWS WAF and AWS ALBs to protect ROSA workloads
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-using-alb-and-waf
+
+toc::[]
+
+// Mobb content metadata
+// Brought into ROSA product docs 2023-09-21
+// ---
+// date: '2021-06-17'
+// title: AWS ALB
+// aliases: ['/docs/aws/waf/cloud-front.md']
+// tags: ["AWS", "ROSA", "OSD"]
+// authors:
+//  - 'Connor Wooley'
+// ---
+
+AWS WAF is a web application firewall that lets you monitor the HTTP and HTTPS requests that are forwarded to your protected web application resources.
+
+You can use an AWS Application Load Balancer (ALB) to add a Web Application Firewall (WAF) to your {product-title} (ROSA) workloads. Using an external solution protects ROSA resources from experiencing denial of service due to handling the WAF.
+
+[NOTE]
+====
+It is recommended that you use the xref:../cloud_experts_tutorials/cloud-experts-using-cloudfront-and-waf.adoc#rosa-mobb-using-cloudfront-and-waf[CloudFront method] unless you absolutely must use an ALB based solution.
```

Comment:
This might fix the build: 

xref:../cloud_experts_tutorials/cloud-experts-using-cloudfront-and-waf.adoc#cloud-experts-using-cloudfront-and-waf[CloudFront method]